yaml/pois.yaml

```diff
@@ -1812,3 +2106,32 @@ filters:
       <<: *output_properties
       kind: attraction
       tier: 6
+
+  # if a more specific shop isn't found, then default to generic shop kind
+  - filter:
+      shop: true
+    min_zoom: 17
+    output:
+      <<: *output_properties
+      kind: shop
```

Comment:
Please ensure test and documentation for `shop` fallback.

Reply:
Added in dcce2ee.

Side note: Most common `shop` values that we don't currently have a `kind` dedicated for:

* `kiosk`
* `confectionery` (although we do match `craft=confectionery`)
* `chemist`
* `beverages`
* `boutique`
* `cosmetics`
* `tailor` (although we do match `craft=tailor`)
* `seafood`
* `farm`
* `interior_decoration`